src/modules/ShortcutGuide/ShortcutGuideModuleInterface/dllmain.cpp

```diff
@@ -185,9 +185,11 @@ class ShortcutGuideModule : public PowertoyModuleIface
     HotkeyEx m_hotkey;
 
     // If the module should be activated through the legacy pressing windows key behavior.
-    const UINT DEFAULT_MILLISECONDS_WIN_KEY_SHOULD_BE_PRESSED = 900;
+    const UINT DEFAULT_MILLISECONDS_WIN_KEY_PRESS_TIME_FOR_GLOBAL_WINDOWS_SHORTCUTS = 1000;
+    const UINT DEFAULT_MILLISECONDS_WIN_KEY_PRESS_TIME_FOR_TASKBAR_ICON_SHORTCUTS = 1000;
```

Comment:
Why did we change the default to 1000 from 900?

Reply:
We can keep 900. I just thought that 1000 is a more natural number. Will change it back to 900.